### ISSUE TITLE
Tracked changes

### ISSUE THREAD

**Issue by fred-atherden** [libero/editor#74]
Changes made to a document by different users will often need to be tracked to allow checking of the edits by publishing staff.

### Author
- [ ] As an author, I want to be able to see edits made to my article by myself, the copy-editor and my fellow authors so that I can check these changes are correct.
- [ ] As an author, I want to be able to **reject edits** made by myself, the copy-editor and my fellow authors so that I can remove those made in error.

### Production staff
- [ ] As a production staff, I want to be able to see edits made to an article by all users and automated tools so that I can check that these changes have been made correctly.
- [ ] As production staff, I want to be able to reject any edit in an article so that I can remove those made in error.
- [ ] As production staff, I want to be accept any edit in an article so that I can finalise the content for publication.
- [ ] As production staff, I want to be able to bulk accept changes so that I can do so as swiftly as possible.
- [ ] As production staff, I want to be able to bulk accept changes for a particular user, so that I can can finalise the content for publication as swiftly as possible.
- [ ] As production staff, I want to be able to bulk accept changes for groups of particular users, so that I can finalise the content for publication as swiftly as possible.

### But what if . . . ?
- 

### Considerations
- 

## XML requirements
```xml
```

Taken from: https://github.com/elifesciences/TextureRequirements/issues/67

**Comment by fred-atherden** [comment]
Note that this ticket requires discussion.

**Comment by NuclearRedeye** [comment]
@FAtherden-eLife Can you double check the above user stories and add any missing please.

**Comment by fred-atherden** [comment]
@NickDuf, I've updated the user stories here (added two to the bottom of the production section). Does it make sense?

**Comment by NickDuf** [comment]
@FAtherden-eLife i think so. This story refers to groups of users I think? e.g. bulk accept vendors and production

As production staff, I want to be able to bulk accept or reject changes for particular users, so that I can finalise the content for publication as swiftly as possible.

**Comment by fred-atherden** [comment]
@NickDuf Yup - exactly. We want to do so for specific users and groups of users. For example:
- Copyeditor
- Copyeditor and an author
- Copyeditor, all authors, pre-editor, production (all users)

**Comment by NickDuf** [comment]
@FAtherden-eLife ok great. Slight adjustment to the story for clarity. Ok with this?

As production staff, I want to be able to bulk accept or reject changes for groups of particular users, so that I can finalise the content for publication as swiftly as possible.

**Comment by fred-atherden** [comment]
Sounds good thanks :+1:
